.github/workflows/build.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       shell: bash
 
 
-    - uses: actions/cache@v3
+    - uses: actions/cache@v2
```

Comment:
Why?

Reply:
I did not intend to modify that. I'll revert it